.gitignore

```diff
@@ -1,4 +1,5 @@
 .coverage
+.cache/
```

Comment:
Was also including it to help with the unit tests storage; will open a separate PR for it.